### ISSUE TITLE
Info in app.js not rendering

### ISSUE THREAD

**Issue by Joseppi83** [codingphasedotcom/hyperapp-starterkit#1]
Hey, Joe.  Thought I fixed this last night, but it bugged out.  I've worked on trying to fix this for about 6 hours total now.  

This is what my browser looks like and console reads when I run "npm run watch."

Just in case you can't read the console, here's that info:

vendor.js:544 Uncaught TypeError: enhancer(...) is not a function
    at createStore (vendor.js:544)
    at createStore (vendor.js:544)
    at vendor.js:916
    at vendor.js:949
    at Array.map (<anonymous>)
    at app (vendor.js:948)
    at Object.<anonymous> (hyper.js:124)
    at __webpack_require__ (vendor.js:55)
    at webpackJsonpCallback (vendor.js:26)
    at hyper.js:1
createStore @ vendor.js:544
createStore @ vendor.js:544
(anonymous) @ vendor.js:916
(anonymous) @ vendor.js:949
app @ vendor.js:948
(anonymous) @ hyper.js:124
__webpack_require__ @ vendor.js:55
webpackJsonpCallback @ vendor.js:26
(anonymous) @ hyper.js:1

Any clue on a fix?
![hyperappscreenshot2](https://user-images.githubusercontent.com/38846965/40317258-cda1932c-5cd5-11e8-9f16-2a04aa8be1db.png)

**Comment by Joseppi83** [comment]
Some other guys on hyperapp's github say that it's due to the big changes in hyperapp's API.

**Comment by baltz1** [comment]
Yeah @Joseppi83  i am getting the same bug here and been trying to find a solution for some time now please keep us updated if you come across a solution as will i if i do.

**Comment by baltz1** [comment]
@Joseppi83  did you end up finding a work around solution?

**Comment by Joseppi83** [comment]
@baltz1  It has to do with the big changes in hyperapp's API after he posted that project.  I skipped it for now.  I'll probably just complete the project in vanilla JS and follow along with the video the best I can.  Definitely want to put that one in my portfolio, too.

The job openings around here aren't really looking for hyperapp experience, so I'm not stuck on getting the hyperapp experience.  Just want to get to a point where I know JS like the back of my hand.  Learning all those frameworks and libraries will be a piece of cake at that point, as long as the documentation is written comprehensively and coherently.

**Comment by baltz1** [comment]
@Joseppi83 it is a really good project i was going to add more features to it but cant due to this bug hopefully there is a work around to this, because i looked into my vender.js and cant understand the function on line 554.

**Comment by Joseppi83** [comment]
@baltz1  Yeah, I hear ya, man.  I spent about 6 hours total trying to debug my errors.  Honestly, I should have just said, "F it," and barreled ahead with HTML, CSS, and vanilla JS right then and there.  Good luck if you continue to try to debug it!  You've always got the bare-bones approach to knock it out.

**Comment by Joseppi83** [comment]
@baltz1 This issue really just boils down to you and I not having enough experience in JavaScript to figure it out.  What we need to do is read the hyperapp documentation, look at how the components are set-up by default, and make the changes necessary to green light the process.

When I finish these other projects and come back to this one, I'm not even going to run the hyperapp-starterkit.  I'm going to give his newest starter kit a shot.  Add hyperapp to it, take the time to read the hyperapp docs, and go from there.  If that doesn't work, it's vanilla JS all the way.

I'll probably get back to this project in 1-2 weeks.  Just depends on how busy I am for work. 

It'd be a better learning experience if we figured this thing out, without reaching out to Joe and saying, "Help me, help me..."

**Comment by Joseppi83** [comment]
@baltz1  He fixed them shits, man.  D-load the new hyperapp starter kit and skip right to the part where he's laying out the project.

**Comment by alanperez** [comment]
@Joseppi83 I'm still getting errors in the console and it still shows a blank page. I downloaded the hyperapp starter kit that was updated on jun 4th.. have you found a work around? https://i.imgur.com/HyHLRkL.png
